doc/api/animation_api.rst

```diff
@@ -32,7 +32,8 @@ to.  If you do not hold a reference to the `Animation` object, it (and
 hence the timers) will be garbage collected which will stop the
 animation.
 
-To save an animation to disk use `Animation.save` or `Animation.to_html5_video`
+To save an animation to disk use `Animation.save`, `Animation.to_html5_video`,
+or `Animation.to_jshtml`.
```

Comment:
From a user persective `to_jshtml` does not save to disk; it returns the generated HTML. Internally, it saves to a temporary directory and loads the html file from there. But that's an implementation detail.